### DISCUSSION TITLE
Join

### DISCUSSION THREAD

**Discussion by BotanicProgrammer** [pmndrs/racing-game#54] [Q&A] [answered]
How can I collaborate? Do I just add a file?  I'm pretty sure that would just make a branch or fork.

**Comment by drcmda** [comment, marked as answer]
Yes, make a branch then a pull request

**Comment by njm222** [comment]
Also checkout [CONTRIBUTING.md](https://github.com/pmndrs/racing-game/blob/main/CONTRIBUTING.md)

**Comment by erwingame** [comment]
Can I join too?
